src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -180,7 +180,7 @@ public async Task GetConfig_ReturnsCorrectAdditionalFile()
             };
             var testSubject = builder.CreateTestSubject();
 
-            var expectedAdditionalFilePath = builder.BindingConfiguration.BuildPathUnderConfigDirectory() + "\\VB\\SonarLint.xml";
+            var expectedAdditionalFilePath = builder.BindingConfiguration.BuildPathUnderConfigDirectory() + "VB\\SonarLint.xml";
```

Comment:
Minor: no project name anymore so BuildPathUnderConfigDirectory already returns something that ends with `\\`